lib/src/generated/bridge_definitions.dart

```diff
@@ -97,16 +86,12 @@ abstract class Rust {
 
   FlutterRustBridgeTaskConstMeta get kBumpFeeTxBuilderFinishConstMeta;
 
-  Future<BdkDescriptor> newDescriptor(
```

Comment:
Would it be possible not to club style changes in the same PR?
It makes checking for changes a little difficult.